crates/sui-json-rpc/src/api.rs

```diff
@@ -233,16 +235,37 @@ pub trait RpcFullNodeReadApi {
         /// the version of the queried object. If None, default to the latest known version
         version: SequenceNumber,
     ) -> RpcResult<GetPastObjectDataResponse>;
+}
+
+#[open_rpc(namespace = "sui", tag = "Governance Read API")]
+#[rpc(server, client, namespace = "sui")]
+pub trait GovernanceReadApi {
+    /// Return all [StakedSui] owned by the `owner` address.
+    #[method(name = "getStakedSui")]
```

Comment:
Rather than having these custom methods, why not just make `getObject` take a filter where I can specify an owner address and an object type? I feel like solving this generically would be a huge win instead of one-off APIs

Reply:
we will have `getObject` with filters as well in later PR.

This method also do bcs deserialisation of the StakedSui object which can't be done generically. We want to make staking easy for exchanges and wallet builders without having to worry about bcs deserialisation or filtering.

Reply:
After the changes @emmazzz made to the StakedSui and Delegation, we can now pair them up in the RPC layer, I will repurpose these getter method to do that.